iotdb/src/test/java/org/apache/iotdb/db/postback/sender/FileManagerTest.java

```diff
@@ -55,20 +56,19 @@ public void setUp() throws Exception {
   }
 
   @After
-  public void tearDown() throws Exception {
+  public void tearDown() throws IOException, InterruptedException {
     Thread.sleep(1000);
-    delete(new File(POST_BACK_DIRECTORY_TEST));
-    new File(POST_BACK_DIRECTORY_TEST).delete();
+    Files.deleteIfExists((java.nio.file.Path) new Path(POST_BACK_DIRECTORY_TEST));
   }
 
-  public void delete(File file) {
+  public void delete(File file) throws IOException {
     if (file.isFile() || file.list().length == 0) {
-      file.delete();
+      Files.deleteIfExists((java.nio.file.Path) new Path(file.getAbsolutePath()));
```

Comment:
why not using import java.nio.file.Path..

Reply:
fixed